src/phoenix/session/evaluation.py

```diff
@@ -155,3 +157,17 @@ def _extract_result(row: "pd.Series[Any]") -> Optional[pb.Evaluation.Result]:
         label=StringValue(value=label) if label else None,
         explanation=StringValue(value=explanation) if explanation else None,
     )
+
+
+def log_evaluations(
+    *evals: Evaluations,
+    endpoint: Optional[str] = None,
+    host: Optional[str] = None,
+    port: Optional[int] = None,
+) -> None:
+    exporter = HttpExporter(endpoint=endpoint, host=host, port=port)
+    for eval in filter(bool, evals):
+        add_evaluations(exporter, eval.dataframe, eval.eval_name)
+    time_limit = time() + 60
+    while time() < time_limit and not exporter._queue.empty():
+        sleep(0.1)
```

Comment:
do we need the limit? Also is it worth having a tqdm progress bar?

Reply:
added `tqdm` and removed `time_limit`